src/seat/keyboard/mod.rs

```diff
@@ -24,8 +24,10 @@ use wayland_client::{
     Attached,
 };
 
+#[rustfmt::skip]
```

Comment:
I'd prefer if those were kept in their respective modules if possible. Is a toplevel `#![rustfmt::skip]` recognized?

Reply:
No. It applies to a following statement or something like that. So that's why they look like that.

Reply:
Alright then.